gui.go

```diff
@@ -634,3 +642,44 @@ func (g *Gui) execKeybindings(v *View, ev *termbox.Event) (matched bool, err err
 	}
 	return matched, nil
 }
+
+func (g *Gui) getTermSize() (int, int, error) {
+	var sz struct {
+		rows uint16
+		cols uint16
+	}
+
+	var termw, termh int
+
+	out, err := os.OpenFile("/dev/tty", os.O_RDWR, 0)
+	if err != nil {
+		return 0, 0, err
+	}
+	defer out.Close()
+
+	signalCh := make(chan os.Signal, 1)
+	signal.Notify(signalCh, syscall.SIGWINCH, syscall.SIGINT)
+
+	for {
+		_, _, _ = syscall.Syscall(syscall.SYS_IOCTL,
+			out.Fd(), uintptr(syscall.TIOCGWINSZ), uintptr(unsafe.Pointer(&sz)))
```

Comment:
Is there a method that does not require unsafe.Pointer?

Reply:
Sorry. I don’t know the other way.
This way used in the termbox. I think this no problem.